web/index.html

```diff
@@ -113,7 +113,10 @@ <h2 class="f4">What does it mean if I get an error?</h2>
                         }
                     }
                     if (!foundAddr) {
-                        outText += "❌ Could not find the given multiaddr in the dht. Instead found: " + Object.keys(respObj.PeerFoundInDHT) + "\n"
+                        outText += "❌ Could not find the given multiaddr in the dht. Instead found:\n"
+						for (const key in respObj.PeerFoundInDHT) {
+							outText += "      " + respObj.PeerFoundInDHT[key] + "\n"

```

Comment:
```suggestion
							outText += "\t" + key + "\n"

```

@momack2 did this code generate the results you saw? The keys in that map are the peers and the value is how many times we saw it. I also switched from spaces to a tab character.